pandas/core/strings.py

```diff
@@ -27,19 +27,19 @@ def _get_array_list(arr, others):
 
 def str_cat(arr, others=None, sep=None, na_rep=None):
     """
-    Concatenate arrays of strings with given separator
+    Concatenate strings in the Series with given separator
 
     Parameters
     ----------
-    arr : list or array-like
     others : list or array, or list of arrays
```

Comment:
Changed to list-like as list is allowed (though we may not distinguish them..?).